src/Samples/Toolkit.SampleApp.Maui/Samples/JobManagerSample.xaml.cs

```diff
@@ -0,0 +1,82 @@
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Tasks.Offline;
+using Esri.ArcGISRuntime.Toolkit;
+using Esri.ArcGISRuntime.Toolkit.Samples;
+
+namespace Toolkit.SampleApp.Maui.Samples
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    [SampleInfo(Category = "JobManager", Description = "Managing jobs across sessions")]
+    public partial class JobManagerSample : ContentPage
+    {
+        public JobManagerSample()
+        {
+            InitializeComponent();
+            JobCollectionView.ItemsSource = JobManager.Shared.Jobs;
+        }
+        private async void GenerateGdb_Click(object sender, EventArgs e)
+        {
+            var url = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/Sync/WildfireSync/FeatureServer");
+            var job = await CreateGenerateGeodatabaseJob(url: url, syncModel: SyncModel.Layer);
+            JobManager.Shared.Jobs.Add(job);
+            job.Start();
+        }
+
+        private async void GenerateMap_Click(object sender, EventArgs e)
+        {
+            var item = await Esri.ArcGISRuntime.Portal.PortalItem.CreateAsync(new Uri("https://www.arcgis.com/home/item.html?id=acc027394bc84c2fb04d1ed317aac674"));
+
+            var map = new Map(item);
+            var naperville = new Envelope(-9813416.487598, 5126112.596989, -9812775.435463, 5127101.526749, SpatialReferences.WebMercator);
+            var job = await CreateOfflineMapJob(map: map, extent: naperville);
+            JobManager.Shared.Jobs.Add(job);
+            job.Start();
+        }
+
+        private async Task<GenerateOfflineMapJob> CreateOfflineMapJob(Map map, Envelope extent)
+        {
+            var task = await OfflineMapTask.CreateAsync(map);
+            var parameters = await task.CreateDefaultGenerateOfflineMapParametersAsync(areaOfInterest: extent);
+            var path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), Guid.NewGuid().ToString());
+            return task.GenerateOfflineMap(parameters, path);
+        }
+
+
+        private async Task<GenerateGeodatabaseJob> CreateGenerateGeodatabaseJob(Uri url, SyncModel syncModel, Envelope? extent = null)
+        {
+            var task = await GeodatabaseSyncTask.CreateAsync(url);
+
+            var parameters = new GenerateGeodatabaseParameters();
+            parameters.Extent = extent ?? task.ServiceInfo?.FullExtent;
+            parameters.OutSpatialReference = extent?.SpatialReference ?? SpatialReferences.WebMercator;
+            parameters.SyncModel = syncModel;
+
+
+            if (syncModel == SyncModel.Layer)
+            {
+                foreach (var info in task.ServiceInfo!.LayerInfos)
+                {
+                    parameters.LayerOptions.Add(new GenerateLayerOption(info.Id));
+                }
+            }
+
+            var path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), new Guid().ToString() + ".geodatabase");
```

Comment:
`new Guid()` is the default constructor -- it's always going to be `00000000-0000-0000-0000-000000000000`.  Did you mean to do [`Guid.NewGuid()`](https://learn.microsoft.com/en-us/dotnet/api/system.guid.newguid?view=net-10.0)?


```suggestion
            var path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), Guid.NewGuid().ToString() + ".geodatabase");
```